format-proposal.md

```diff
@@ -0,0 +1,128 @@
+# Format proposal
+
+- **PRE: parameters and variables share their namespace**
+
+Parameters of any kind:
+
+```asp
+parameter_value("a",value,1).                   % single integer
```

Comment:
I renamed to parameter_value. Do you need the parameter type? Can we just use type for both variables and parameters?

Reply:
Thx.
Parameter type is not needed.

Reply:
And please no `variable_type` for parameters. Then its hard to distinguish them from variables.